docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -55,43 +74,43 @@ these options:
+Balancer (NLB) to handle incoming traffic.
 
-You can use AWS Certificate Manager to handle TLS termination with AWS-managed
-certificates.
+#### Using AWS Certificate Manager
 
-You should be aware of the limitations for using AWS Certificate Manager to
-provision TLS credentials for Teleport:
+You can use AWS Certificate Manager to handle TLS termination with AWS-managed certificates.
 
-- This will prevent the Teleport Application Service from working via CLI using
-  client certificates. Application access will still work via a browser.
-- Command-line application access does not work with ACM. Using ACM will prevent
-  Teleport from facilitating application access via CLI (using client
-  certificates), as Teleport will not be handling its own TLS termination.
-- Using ACM through a AWS Load Balancer prevents the required traffic for
-  Postgres or MongoDB through Teleport's web port. If you choose to use the ACM
-  approach, we will show you how to configure a separate listener for Postgres
-  or MongoDB.
+This method uses a Kubernetes `Ingress`, which can provision an underlying AWS Application Load
+Balancer (ALB) to handle incoming traffic if one does not already exist. It also requires the
+installation and setup of the AWS Load Balancer controller.
 
-<Notice type="warning">
+You should be aware of these potential limitations when using Layer 7 load balancers with Teleport:
 
-If you would like the Teleport Application Service and Database Service to
-function as expected, you should use the `cert-manager` approach unless there is
-a specific reason to use ACM.
+- Connecting to Kubernetes clusters at the command line requires the use of the `tsh proxy kube` or
+  `tsh kubectl` commands and `tsh proxy db`/`tsh db connect` commands respectively. It is not
+  possible to connect `kubectl` directly to Teleport listeners without the use of `tsh` as a proxy client
+  in this mode.
+- Connecting to databases at the command line requires the use of the `tsh proxy db` or `tsh db connect`
+  commands. It is not possible to connect database clients directly to Teleport listeners without the use of `tsh`
+  as a proxy client in this mode.
+- The use of Teleport and `tsh` v13 or higher is required.
 
-</Notice>
+<Admonition type="warning">
+  Using ACM with an ALB also requires that your cluster has a fully functional installation of the AWS Load Balancer
+  controller with required IAM permissions. This guide provides more details below.
+</Admonition>
 
 #### Using your own TLS credentials
 
 With this approach, you are responsible for determining how to obtain a TLS
 certificate and private key for your Teleport cluster, and for renewing your
 credentials periodically. Use this approach if you would like to use a trusted
 internal certificate authority instead of Let's Encrypt or AWS Certificate
-Manager.
+Manager. This method uses a Kubernetes `LoadBalancer` and will provision an
+underlying AWS NLB.
 
```

Comment:
I think a h3 section is missing here, else we fall into the h4 "Using your own TLS"